src/components/Explore/SpeciesView.js

```diff
@@ -67,11 +73,42 @@ const SpeciesView = ( {
     {
       ...queryParams,
       fields: {
-        taxon: Taxon.TAXON_FIELDS
+        taxon: Taxon.SCORE_IMAGE_FIELDS
       }
     }
   );
 
+  const taxonIds = data.map( r => r.taxon.id );
+
+  const { data: seenByCurrentUser } = useAuthenticatedQuery(
+    ["fetchSpeciesCounts", taxonIds],
+    ( ) => fetchSpeciesCounts( {
+      user_id: currentUser?.id,
+      taxon_id: taxonIds,
+      fields: {
+        taxon: {
+          id: true
+        }
+      }
+    } ),
+    {
+      enabled: !!( taxonIds.length > 0 && currentUser )
+    }
+  );
+
+  const taxonIdsSeenByCurrentUser = useMemo( ( ) => seenByCurrentUser?.results?.map(
+    r => r.taxon.id
+  ) || [], [seenByCurrentUser?.results] );
+
+  useEffect( ( ) => {
+    if ( taxonIdsSeenByCurrentUser.length > 0 ) {
+      const uniqueSeenList = _.uniq( taxonSeen.concat( taxonIdsSeenByCurrentUser ) );
+      if ( uniqueSeenList.length > taxonSeen.length ) {
+        setTaxonSeen( uniqueSeenList );
+      }
```

Comment:
If you're maintaining a unique list of numbers, you probably want to use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) instead of an array. That way you don't need to use `_.uniq()`. 

Also, `taxonSeen` sounds like it's either a taxon or a boolean. Since it is instead a set of IDs of taxa observed by the current user, a better name might be something like `observedTaxonIds`.